### PR TITLE
version upgradation to 1.4.1

### PR DESCRIPTION
**Pull request by CharmisG**
- Upgraded get started doc
- Upgraded SDK version to 1.6.1 for android and iOS
- Upgrade minSDK to 24 and compileSDK and targetSDK version to 30
- convert from java to kotlin (quick start folder)

### CODE REVIEW

**Comment by hukid** [comment]
@CharmisG can you add description about this PR?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.4.1 Sept 06, 2022
+
+* Bump the minimum Reader SDK version support to 1.6.1 for iOS and 1.5.5 for Android
```
Comment (hukid) [inline CHANGELOG.md:5]: Why only 1.5.5 for Android?

Comment (CharmisG) [reply CHANGELOG.md]: Changed to 1.6.1 for android

**Comment by CharmisG** [comment]
It's new release update :
- upgraded get started doc
- Upgraded SDK version to 1.6.1 for android and iOS
- Upgrade minSDK to 24 and compileSDK and targetSDK version to 30

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.4.1 Sept 06, 2022
+
+* Bump the minimum Reader SDK version support to 1.6.1 for iOS and 1.6.1 for Android
+* Update application UI
```
Comment (hukid) [inline CHANGELOG.md:6]: What application? can we be more specific? I believe that's the quick start example UI?

**Review by hukid — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (hukid) [reply CHANGELOG.md:6]: - [ ] update "Update application UI" to be more specific

Comment (CharmisG) [reply CHANGELOG.md]: Done

File: docs/get-started.md [hunk 3]
```diff
@@ -237,28 +238,26 @@ installing Reader SDK for iOS, see [Reader SDK iOS Setup Guide].
    * `NSPhotoLibraryUsageDescription` : This app integrates with Square for card
       processing. Upload your account logo, feature photo and product images
       with the photos stored on your mobile device.
+
 1. Update the `application:didFinishLaunchingWithOptions:` method in your app
```
Comment (hukid) [inline docs/get-started.md:242]: Is this still the same method name? should it be `func application`?

File: reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+         * initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
+         *
+         * @param context
+         * @param reactInstanceManager
+         */
+        private fun initializeFlipper(
+                context: Context, reactInstanceManager: ReactInstanceManager) {
+            if (BuildConfig.DEBUG) {
+                try {
+                    /*
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt:70]: nit: please format the comment with the correct indents.

**Review by hukid — APPROVED** [review]

File: docs/get-started.md (hunk 3 above)
Comment (CharmisG) [reply docs/get-started.md:242]: Done

File: reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt (hunk 4 above)
Comment (CharmisG) [reply reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt:70]: done

Comment (CharmisG) [reply reader-sdk-react-native-quickstart/android/app/src/main/java/com/rnreadersdksample/MainApplication.kt:70]: done

**Pull request merged**